src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md

```diff
@@ -0,0 +1,102 @@
+---
+group: extension-best-practices
+title: Create Access Control List(ACL) Rule
+contributor_name: Ziffity
+contributor_link: https://www.Ziffity.com/
+---
+
+Access Control List(ACL) rules allow admin to limit the permissions of users in Magento. We can use ACL rules for authorize the users to access menu,controllers and API endpoints.
+
```

Comment:
You need to explicitly state what you'll be doing in this example. You're creating three custom resources (Custom Menu, Create, Delete), creating a role that has access to these resources, and taking steps to restrict access by three entities (Admin users, controllers, and web APIs).